### DISCUSSION TITLE
How do I submit a config file for an existing device?

### DISCUSSION THREAD

**Discussion by cornellrwilliams** [zwave-js/zwave-js#5942] [General]
The Zooz Zen17 had a fw update that added new parameters. I created the config file and tested that it works but have no clue how to submit it. I've attached the file below if anyone knows how and wants to help.

[zen17.txt](https://github.com/zwave-js/node-zwave-js/files/11860068/zen17.txt)

![Zen17 Config](https://github.com/zwave-js/node-zwave-js/assets/92619869/0d2c9331-4828-4422-bdf9-f1a035a86a2e)

**Comment by AlCalzone** [comment]
The contributing process is roughly as follows:
1. [Fork the repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository) (make your own copy) if you haven't already.
2. [Create a branch](https://docs.github.com/en/get-started/quickstart/github-flow#create-a-branch) for your changes.
3. Make the changes there, and commit the changes to your branch. 
4. [Create a pull request](https://docs.github.com/en/get-started/quickstart/github-flow#create-a-pull-request) so we can include the changes.

You can either work locally, which requires you to set up software, or you can use a cloud-hosted VSCode that's fully set up for this project https://zwave-js.github.io/node-zwave-js/#/config-files/contributing-files?id=editor-setup - and which also helps you with the git-related stuff a bit.
The advantage of this is that we have automated checks in place to make sure the file is correct.
